data/blog/nostr-grants-august-2024.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: "Sixth Wave of Nostr Grants"
+date: '2024-08-08'
+tags: ['OpenSats', 'grants', 'nostr', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/TODO']
```

Comment:
Make sure to set the image here once you upload it to the `/blog` folder, like this:

```suggestion
images: ['/static/images/blog/45-nostr-grants-august-2024.jpg']
```